kiianigen.js

```diff
@@ -136,8 +138,11 @@ function main() {
     }
 
     // Get info about the current configuration files from KType-Standard directory.
-    json = JSON.parse(require('fs')
-                          .readFileSync(srcConfigDir + '/KType-Standard.json', 'utf8'));
+    if(!fs.existsSync(srcConfigDir + KTYPE_FILE)) {
+        console.log("KType-Standard.json not found")
```

Comment:
Should this kill the script at this point in addition to the log statement?

Reply:
it probably should, but this was the narrowest pull request just to emit a more useful error then the stack trace.